C/plugins/north/OMF/plugin.cpp

```diff
@@ -980,6 +980,15 @@ uint32_t plugin_send(const PLUGIN_HANDLE handle,
 		connInfo->omfversion = EDS_OMF_VERSION;
 	}
 #endif
+
+// Version for Connector Relay is 1.0 only.
+#ifdef CR_OMF_VERSION
```

Comment:
I don't believe the connector relay ever went beyond OMF 1.0, so we probably do not need to have the ifdef here.

Reply:
Done